rdamsc/records.py

```diff
@@ -2051,6 +2051,7 @@ class Scheme(Record):
     schema = {
         "title": {"type": "text", "useful": True},
         "description": {"type": "html", "useful": True},
+        "citation_docs": {"type": "html", "useful": True},
```

Comment:
"useful" might be the wrong word here; perhaps I should have used "requisite". I didn't want to use "required" since none of the fields are technically required.

"useful" is short for the statement "The record cannot be considered useful without a value for this field."
"optional" is short for the statement "The record can be considered complete even without a value for this field."

I think "optional" is probably the right flag to use.